tools/autograd/derivatives.yaml

```diff
@@ -736,11 +736,11 @@
   self: value_selecting_reduction_backward(grad, dim, indices, self.sizes(), keepdim)
 
 - name: mul.Tensor(Tensor self, Tensor other) -> Tensor
-  self: grad * other
-  other: grad * self
+  self: grad * other.conj()
```

Comment:
This will fail like the scalar version if self and other are not both real or both complex right?

Reply:
yeah updated to check for R-> C. this is being used in `sin` and `cos` gradgradcheck. will add more specific tests in follow up PR.